src/chains/ethereum/ethereum/src/database.ts

```diff
@@ -70,31 +87,28 @@ export default class Database extends Emittery {
       }
       this.directory = directory;
 
-      // specify an empty `prefix` for browser-based leveldown (level-js)
-      const leveldownOpts = { prefix: "" };
-      const store = encode(leveldown(directory, leveldownOpts), levelupOptions);
-      this.#rootStore = store;
-      db = levelup(store);
+      const store = <GanacheLevel>new Level(directory, LEVEL_OPTIONS);
+      db = store;
     }
 
     // don't continue if we closed while we were waiting for the db
     if (this.#closed) return this.#cleanup();
 
     const open = db.open();
-    this.trie = sub(db, "T", levelupOptions);
+    this.trie = <GanacheSublevel>db.sublevel("T", LEVEL_OPTIONS);
 
     this.db = db;
     await open;
 
     // don't continue if we closed while we were waiting for it to open
     if (this.#closed) return this.#cleanup();
 
-    this.blocks = sub(db, "b", levelupOptions);
-    this.blockIndexes = sub(db, "i", levelupOptions);
-    this.blockLogs = sub(db, "l", levelupOptions);
-    this.transactions = sub(db, "t", levelupOptions);
-    this.transactionReceipts = sub(db, "r", levelupOptions);
-    this.storageKeys = sub(db, "s", levelupOptions);
+    this.blocks = <GanacheSublevel>db.sublevel("b", LEVEL_OPTIONS);
+    this.blockIndexes = <GanacheSublevel>db.sublevel("i", LEVEL_OPTIONS);
+    this.blockLogs = <GanacheSublevel>db.sublevel("l", LEVEL_OPTIONS);
+    this.transactions = <GanacheSublevel>db.sublevel("t", LEVEL_OPTIONS);
+    this.transactionReceipts = <GanacheSublevel>db.sublevel("r", LEVEL_OPTIONS);
+    this.storageKeys = <GanacheSublevel>db.sublevel("s", LEVEL_OPTIONS);
```

Comment:
```suggestion
    this.blocks = db.sublevel("b", LEVEL_OPTIONS) as GanacheSublevel;
    this.blockIndexes = db.sublevel("i", LEVEL_OPTIONS) as GanacheSublevel;
    this.blockLogs = db.sublevel("l", LEVEL_OPTIONS) as GanacheSublevel;
    this.transactions = db.sublevel("t", LEVEL_OPTIONS) as GanacheSublevel;
    this.transactionReceipts = db.sublevel("r", LEVEL_OPTIONS) as GanacheSublevel;
    this.storageKeys = db.sublevel("s", LEVEL_OPTIONS) as GanacheSublevel;
```
Multiline suggestion FTW! https://github.blog/changelog/2019-10-01-multi-line-references-and-comments-beta/